### PR TITLE
Add <ins> and <del> tags appropriately.

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
Based on feedback in #102.

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -514,26 +524,8 @@ <h1>Runtime Semantics: Evaluation</h1>
...
-          1. If Type(_func_) is not Object, throw a *TypeError* exception.
-          1. If IsCallable(_func_) is *false*, throw a *TypeError* exception.
-          1. If _tailPosition_ is *true*, perform PrepareForTailCall().
-          1. Let _result_ be Call(_func_, _thisValue_, _argList_).
-          1. Assert: If _tailPosition_ is *true*, the above call will not return here, but instead evaluation will continue as if the following return has already occurred.
-          1. Assert: If _result_ is not an abrupt completion, then Type(_result_) is an ECMAScript language type.
-          1. Return _result_.
-        </emu-alg>
+
+        <p><i>(not modified)</i></p>
```
Comment (ljharb) [inline spec.html]: If this isn’t modified, should the whole emu-clause be removed?

Comment (DanielRosenwasser) [reply spec.html]: I assumed so, will do so now.

File: spec.html [hunk 2]
```diff
@@ -422,22 +422,31 @@ <h1>Runtime Semantics: Evaluation</h1>
         <emu-alg>
```
Comment (jridgewell) [inline spec.html]: I don't think any of these changes belongs in this proposal (unless there's some necessary change I'm not seeing). Spec refactors would be better as a editorial change on ecma262 repo.

Comment (DanielRosenwasser) [reply spec.html]: Which changes did you have in mind?

There are a couple of refactors, but only with the express purpose of reuse among new clauses. For example, EvaluateDynamicPropertyAccess is used in the runtime semantics for the following production.

> *OptionalChain* : *OptionalChain* `[` *Expression* `]`</emu-grammar>

Similarly, EvaluateStaticPropertyAccess is used in the runtime semantics for

> *OptionalChain* : `?.` *IdentifierName*

Comment (jridgewell) [reply spec.html]: Ahhh, now it makes more sense. 👍, let's keep this refactoring.

**Pull request merged**